README.md

```diff
@@ -135,9 +136,12 @@ _NOT_ configures `fluentd` to log to the file. See Example Playbook for how.
   roles:
     - ansible-role-fluentd
   vars:
-    fluentd_flags: "{% if ansible_os_family == 'FreeBSD' %}-p {{ fluentd_plugin_dir }}{% elif ansible_os_family == 'Debian' %}{% elif ansible_os_family == 'RedHat' %}{% elif ansible_os_family == 'OpenBSD' %}--daemon /var/run/fluentd/fluentd.pid --config /etc/fluentd/fluent.conf -p /etc/fluentd/plugin{% endif %}"
+    fluentd_extra_groups:
```

Comment:
the notation is different from `fluentd_extra_groups: tty,bin` which is used in `tests/serverspec/default.yml`.

Reply:
fixed in aea14ec